### PR TITLE
#260: Add alias namespace

### LINKED ISSUE #260: Alias namespace to magistrate

### PR DESCRIPTION
**Pull request by thearusable**
Closes #260

### CODE REVIEW

**Review by cz4rs — APPROVED** [review]
Looks good to me.

File: src/checkpoint/checkpoint.h [hunk 1]
```diff
@@ -73,4 +73,7 @@
 #include "checkpoint/checkpoint_api.h"
 #include "checkpoint/checkpoint_api.impl.h"
 
+// Namespace alias for the old name of the library
+namespace checkpoint = magistrate;
+
 #endif /*INCLUDED_CHECKPOINT_CHECKPOINT_H*/
```
Comment (lifflander) [inline src/checkpoint/checkpoint.h:79]: We should just the header guards to correspond to the namespace change.

**Review by lifflander — CHANGES_REQUESTED** [review]

**Review by lifflander — COMMENTED** [review]
Let's just add the alias the other way around.

**Comment by thearusable** [comment]
I updated the implementation to not change the name of checkpoint namespace. I needed to rename the magistrate namespace in examples.

**Review by lifflander — APPROVED** [review]

**Pull request merged**